### ISSUE TITLE
Search action links can't be activated by keypress

### ISSUE THREAD

**Issue by roblourens** [Microsoft/vscode#59774]
- Do text search that returns no results
- Focus "Open settings" by tabbing
- Press space or enter
- It should activate the link, but nothing happens

**Comment by itamark** [comment]
Hey @roblourens I'm trying to reproduce this but OSS search seems to be taking forever. Have you ever had this happen?

**Comment by roblourens** [comment]
Make sure you've run yarn after pulling the latest

**Comment by itamark** [comment]
yeah that was it - ripgrep had issues and I had to upgrade yarn version.
